scripts/src/services/users.ts

```diff
@@ -0,0 +1,54 @@
+import * as jsonwebtoken from "jsonwebtoken";
+
+type JWTClaims = {
+	iss: string; // issuer
+	exp: number; // expiration
+	iat: number; // issued at
+	sub: string; // subject
+}
+
+type JWTContent = {
+	header: {
+		typ: string;
+		alg: string;
+		key: string;
+	};
+	payload: JWTClaims & {
+		// custom claims
+		user_id: string;
+	};
+	signature: string;
+}
+
+function getApplicationPublicKey(application_id: string, key_id: string) {
+	// return the public key for the given application.
+	// an application might have multiple keys. each key identified by key_id.
+
+	const publicKeys = {
```

Comment:
I think it's better to put this in another file and load (import) it

Reply:
for sure - it will be loaded from the DB and cached in memory